.github/workflows/weekly.yaml

```diff
@@ -26,6 +26,7 @@ jobs:
           private_key: ${{ secrets.TRUNK_OPEN_PR_APP_PRIVATE_KEY }}
 
       - name: Trunk Upgrade
+        id: upgrade
```

Comment:
Revert? Assuming you used this for the test run.

Reply:
Leaving in. Doesn't detract, and makes future debugging easier